### PR TITLE
fix: accessing first element of empty list fails on Terraform 1.3

### PR DESCRIPTION
**Pull request by greut**
:bug:

First level:
- https://github.com/edgelaboratories/dawn-consul-server/pull/233
- https://github.com/edgelaboratories/dawn-vault/pull/267
- https://github.com/edgelaboratories/dawn-nomad-server/pull/207

Second level:
- https://github.com/edgelaboratories/dawn-tests/pull/153

Third level:
- https://github.com/edgelaboratories/dawn-opensearch/pull/105

### CODE REVIEW

File: target-group.tf [hunk 1]
```diff
@@ -3,7 +3,10 @@ locals {
 
   target_group_arns = concat(
     var.target_group_arns,
-    local.target_group_enabled == true ? [aws_lb_target_group.this[0].arn] : []
+    [
+      for _, target_group in aws_lb_target_group.this :
```
Comment (ducretje-evooq) [inline target-group.tf:7]: 👀 never saw this syntax, sorry for the potato question but I don't understand the `_,`?

Comment (greut) [reply target-group.tf:7]: `_` means ignore, usually. cf https://medium.com/python-features/naming-conventions-with-underscores-in-python-791251ac7097#cf63

**Comment by greut** [comment]
@edgelaboratories/infra

**Review by vthiery — APPROVED** [review]

**Pull request merged**